packages/assets-controllers/src/NftDetectionController.test.ts

```diff
@@ -264,7 +276,6 @@ describe('NftDetectionController', () => {
           },
         ],
       });
-    console.log(nock.activeMocks());
```

Comment:
I am not sure why we were doing this; this was just adding to the noise when running these tests.

Reply:
I guess someone forgot to remove it while debugging